src/pages/Beta/Swap/PairInfo/PairChart/index.tsx

```diff
@@ -55,7 +54,7 @@ const PairChart: React.FC<Props> = ({ pair, tokenA, tokenB }) => {
       ? pairTokensChartData[1]
       : []
 
-  const coingeckoData = useCoingeckoChartData(tokenA || WAVAX[chainId], tokenB || PNG[chainId])
+  const coingeckoData = useCoingeckoChartData(tokenA || WAVAX[chainId])
```

Comment:
1. what is the reason to change this logic ? I didn't quite understand it 
2. there is issue with text color
![Screenshot 2022-07-26 at 5 06 22 PM](https://user-images.githubusercontent.com/6604146/180996983-d7695d57-0140-465e-a70e-78a7ea80d7cd.png)